### PR TITLE
Update Python 3.7 syntax to 3.8

### PR DESCRIPTION
**Pull request by kloczek**
**Pull request**
Filter all code over `pyupgrade --py38-plus`.

**How Has This Been Tested?**
I;m using my typical packaging procedure:
- `python3 -sBm build -w --no-isolation`
- because I'm calling `build` with `--no-isolation` I'm using during all processes only locally installed modules
- install .whl file in </install/prefix> using `installer` module
- run pytest with $PYTHONPATH pointing to sitearch and sitelib inside </install/prefix>
- build is performed in env which is *`cut off from access to the public network`* (pytest is executed with `-m "not network"`)

**Checklist**

- [x] I have read [CONTRIBUTING.md](https://github.com/pdfminer/pdfminer.six/blob/master/CONTRIBUTING.md). 
- [ ] I have added a concise human-readable description of the change to [CHANGELOG.md](https://github.com/pdfminer/pdfminer.six/blob/master/CHANGELOG.md).
- [x] I have tested that this fix is effective or that this feature works.
- [ ] I have added docstrings to newly created methods and classes.
- [ ] I have updated the [README.md](https://github.com/pdfminer/pdfminer.six/blob/master/README.md) and the [readthedocs](https://github.com/pdfminer/pdfminer.six/tree/master/docs/source) documentation. Or verified that this is not necessary.

### CODE REVIEW

File: pdfminer/converter.py [hunk 1]
```diff
@@ -783,7 +783,7 @@ def render(item: LTItem) -> None:
                 )
                 self.write(s)
             elif isinstance(item, LTFigure):
-                s = '<figure name="%s" bbox="%s">\n' % (item.name, bbox2str(item.bbox))
+                s = '<figure name="{}" bbox="{}">\n'.format(item.name, bbox2str(item.bbox))
```
Comment (pietermarsman) [inline pdfminer/converter.py:786]: What do you think of using an f-string here as well? 

And in the other locations where `.format()` is used.

**Review by pietermarsman — CHANGES_REQUESTED** [review]
Love it! 

Only one comment: what do you think of using f-strings everywhere? I know [pyupgrade is intentionally timid](https://github.com/asottile/pyupgrade?tab=readme-ov-file#f-strings) when converting to f-strings, but I think it is more readable in our almost all of our cases.

**Comment by pietermarsman** [comment]
@kloczek Thanks for your contribution!

**Comment by kloczek** [comment]
Thank you

**Pull request merged**